scripts/create_pelorus_operator

```diff
@@ -158,17 +158,19 @@ elif [ "$(ls -A "${patches_dir}/bundle-patches"/*.diff 2>/dev/null)" ]; then
         done
 fi
 
-# Correct operator version, to be latest +1
-OPERATOR_VERSION=$(curl -H "Authorization: Bearer XYZ" -X GET "https://quay.io/api/v1/repository/${OPERATOR_ORG_NAME}/${OPERATOR_PROJECT_NAME}/tag/" | jq .tags[].name | head -1 | sed -e 's|\"||g')
+# Print operator versions
 echo "INFO: Current operator version: ${OPERATOR_VERSION}"
-NEW_OPERATOR_VERSION=$(echo ${OPERATOR_VERSION} | awk -F. -v OFS=. '{$NF += 1 ; print}')
 echo "INFO: New operator version: ${NEW_OPERATOR_VERSION}"
```

Comment:
It would be better to have a single echo with something like `"Operator was bumped from ${OPERATOR_VERSION} to ${NEW_OPERATOR_VERSION} version"`?

Reply:
Maybe, it's just helper script. If we want the nice formatting etc, we would use printf() in bash.